src/app/components/LiteSiteCta/index.tsx

```diff
@@ -0,0 +1,104 @@
+/** @jsx jsx */
+import { useContext } from 'react';
+import { jsx } from '@emotion/react';
+import Paragraph from '../Paragraph';
+import Text from '../Text';
+import { LeftChevron, RightChevron } from '../icons';
+import { ServiceContext } from '../../contexts/ServiceContext';
+import { RequestContext } from '../../contexts/RequestContext';
+import styles from './index.styles';
+import { defaultTranslations } from './liteSiteConfig';
+
+type CtaLinkProps = {
+  isRtl: boolean;
+  href: string;
+  text: string;
+  fontVariant?: string;
+  showChevron?: boolean;
+  ignoreLiteExtension?: boolean;
+  className?: string;
+};
+
+const CtaLink = ({
+  isRtl,
+  href,
+  text,
+  fontVariant = 'sansRegular',
+  showChevron = false,
+  ignoreLiteExtension = false,
+  className,
+}: CtaLinkProps) => {
+  const chevron = isRtl ? (
+    <LeftChevron css={styles.chevron} />
+  ) : (
+    <RightChevron css={styles.chevron} />
+  );
+
+  return (
+    <a
+      href={href}
+      className={className}
+      css={styles.link}
+      {...(ignoreLiteExtension && { 'data-ignore-lite': true })}
+    >
+      <Text size="brevier" fontVariant={fontVariant} css={styles.linkText}>
+        {text}
+      </Text>
+      {showChevron && chevron}
+    </a>
+  );
+};
+
+const LiteSiteCta = () => {
+  const { dir, translations } = useContext(ServiceContext);
+  const { canonicalLink } = useContext(RequestContext);
+  const isRtl = dir === 'rtl';
+  const { liteSite = defaultTranslations } = translations;
+  const {
+    onboardingMessage,
+    toMainSite,
+    informationPage,
+    informationPageLink,
+    dataSaving,
+  } = liteSite;
+  const id = 'LiteSiteCta';
+
+  return (
+    <section
+      role="region"
+      data-e2e="lite-cta"
+      aria-labelledby={id}
+      css={styles.outerContainer}
+    >
+      <Text as="strong" id={id} hidden>
+        {dataSaving}
+      </Text>
+      <div css={styles.container}>
+        <Paragraph size="brevier" css={styles.message}>
+          {onboardingMessage}
+        </Paragraph>
+        <Paragraph data-e2e="to-main-site">
```

Comment:
Is this data-e2e required (I can't see it used anywhere)

Reply:
It's used by the[ e2e tests here](https://github.com/bbc/simorgh/blob/ef7b4c2253061d78321e7ce3493db16826ba8ad6/cypress/e2e/pages/articles/testsForLiteOnly.js#L9). I thought it would be better to give them explicit data-e2e attributes rather than something like "select first/second paragraph" in Cypress. But is there a better way?

Reply:
Oops my apologies, I didn't see those tests. The variable names are perfect, because they describe exactly what it does.